src/main/java/com/internet/shop/controllers/InjectDataController.java

```diff
@@ -0,0 +1,39 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import com.internet.shop.service.ProductService;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class InjectDataController extends HttpServlet {
+    public static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+    private final ShoppingCartService shoppingCart = (ShoppingCartService) injector
```

Comment:
why this service is final and others are not? 
keep your code consistent: make all of them final, or non final